.github/workflows/dart.yml

```diff
@@ -16,7 +16,7 @@ env:
 
 jobs:
   job_001:
-    name: "OS: linux; SDK: dev; PKG: mono_repo; TASKS: `cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate`"
+    name: "linux; Dart dev; PKG: mono_repo; `cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate`"
```

Comment:
I think the `TASKS:` header in front was kind of nice to separate these sections

Reply:
I was thinking this is always the last thing, content always in ticks – these get super long and it's easy to miss the content.

So "kinda nice" or "required/essential"?

Reply:
I agree with omitting the label to save space.

Reply:
Ya I don't really think its going to help much because the content is so long for this part, but 🤷 

I am also ok with just the `;`  separators